src/main/java/com/productdock/library/inventory/book/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.productdock.library.inventory.book;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository extends MongoRepository<BookEntity, String> {
```

Comment:
This name to, basically we need better names. It is not about storing the book, it is about inventory perspective